File: code/project-java-copy/project-java-copy/src/Element.java
class Element {
	final int tid ;   
	final int iutils; 
	final int rutils; 
	
	public Element(int tid, int iutils, int rutils){
		this.tid = tid;
		this.iutils = iutils;
		this.rutils = rutils;
	}
	
	public void print(int depth){
		for (int i=0;i<depth;i++) System.out.print("\t");
		System.out.println("\t"+tid+" iutils: "+iutils+" rutils: "+rutils);
	}
}

File: code/TKO_CSharp/TKO_CSharp/Models/Element.cs
namespace TKO_CSharp.Models
{
    public class Element
    {
        public int TId { get; set; }
        public int Iutils { get; set; }
        public int Rutils { get; set; }

        public Element(int tid, int iutils, int rutils)
        {
            TId = tid;
            Iutils = iutils;
            Rutils = rutils;
        }
    }
}
